src/global-styles/base.css

```
@@ -1,6 +1,73 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import "tailwindcss";
+
+@plugin "@hasparus/tailwind/scrollview-fade";
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@plugin`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)